app/views/content/non-uk-teachers/international-qualified-teacher-status.md

```diff
@@ -20,49 +20,45 @@ keywords:
     - Qualified teacher status
 ---
 
-International qualified teacher status (iQTS) is a teaching qualification backed by the UK government. iQTS meets the same high standards as [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts), and leads to the automatic award of QTS. You can train where you live and work, with no need to visit England.
+International qualified teacher status (iQTS) is a teaching qualification that's the equivalent of [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). iQTS enables you to train to teach in the country where you live and work, with no need to visit England.
 
 ## Why choose iQTS?
 
-iQTS is offered by world-class English teacher training providers working with schools in countries around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region, and get in-person support from a trained mentor.
+iQTS is offered by teacher training providers in England that work with schools around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region. You'll also get in-person support from a trained mentor.
```

Comment:
Should this be, 'experience in a school in your country'? We say country earlier in the sentence so should it be consistent?